src/views/lighttable.c

```diff
@@ -390,6 +393,15 @@ static void _view_lighttable_collection_listener_callback(gpointer instance, gpo
   _unregister_custom_image_order_drag_n_drop(self);
   _register_custom_image_order_drag_n_drop(self);
 
+  // in filemanager, we want to reset the offset to the beggining
```

Comment:
This seems to be causing #2320, I also have it when editing an image and go back to lighttable.

Reply:
I can't reproduce, neither with copy-pasting history or with switching back to lt from dr... Do you have something special in your collect queries ? or filters ?
Because this "reset" happens only when the collection is changed, and I don't see why this signal would happen in this cases...